documentation/docs/04-hooks.md

```diff
@@ -73,9 +73,9 @@ export function getSession({ context }) {
 
 ### handle
 
-This function runs on every request, and determines the response. The second argument, `render`, calls SvelteKit's default renderer. This allows you to modify response headers or bodies, or bypass SvelteKit entirely (for implementing endpoints programmatically, for example).
+This function runs on every request, and determines the response. It receives the `request` and `render`, which calls SvelteKit's default renderer. This allows you to modify response headers or bodies, or bypass SvelteKit entirely (for implementing endpoints programmatically, for example).
```

Comment:
coming back to this a bit later I'm not sure I still love my original suggestion. how about tweaking it a bit more?
```suggestion
This function runs on every request, and determines the response. It receives the `request` object and `render` method, which calls SvelteKit's default renderer. This allows you to modify response headers or bodies, or bypass SvelteKit entirely (for implementing endpoints programmatically, for example).
```

Reply:
Yeah, I like that. I was trying to think of a way to make it clearer that the ", which" only applied to the second thing, and I think this does a good enough job.